hono-router.hbs

```diff
@@ -59,7 +59,7 @@ const validationService = new ValidationService(models);
 
 // WARNING: This file was auto-generated with tsoa. Please do not modify it. Re-run tsoa to re-generate this file: https://github.com/lukeautry/tsoa
 
-export function RegisterRoutes<T = any>(router: Hono<T>) {
+export function RegisterRoutes<T = any>(router: Hono<any>) {
```

Comment:
What is the compilation error you get with passing T here? I can't replicate the issue..

Reply:
https://github.com/markusahlstrand/tsoa-hono/pull/1#pullrequestreview-1718930319 see this PR! You can see where the CI/CD is failing

These are the changes I made locally to get TypeScript to be happy